website/docs/howto.md

```diff
@@ -0,0 +1,278 @@
+---
+id: howto
+title: How to use Gatekeeper
+---
+
+Gatekeeper uses the [OPA Constraint Framework](https://github.com/open-policy-agent/frameworks/tree/master/constraint) to describe and enforce policy. Look there for more detailed information on their semantics and advanced usage.
+
+### Constraint Templates
```

Comment:
Should we adjust heading levels in this doc?

Reply:
updated